pkgs/development/tools/phase/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq nix-prefetch common-updater-scripts nix coreutils
+# shellcheck shell=bash
+set -euo pipefail
+
+# Fetch the latest Phase CLI version from GitHub https://github.com/phasehq/cli
+REPO="phasehq/cli"
+RELEASE_TAG=$(curl -s "https://api.github.com/repos/$REPO/releases/latest" | jq -r .tag_name)
+VERSION=${RELEASE_TAG#v}
+
+echo "Latest Phase CLI release: $VERSION"
+
+# Prepare x86 and arm64 targets for Linux and MacOS
+ARCHS=(
+    "x86_64-linux:linux_amd64"
+    "x86_64-darwin:macos_amd64"
+    "aarch64-linux:linux_arm64"
+    "aarch64-darwin:macos_arm64"
+)
+
+# Paths to default.nix and SRI (Sub-resource integrity) file
+NIXCONFIG=pkgs/development/tools/phase/default.nix
+SRIFILE=pkgs/development/tools/phase/hashes.json
+
+# Clear file from a previous run
+rm -f "$SRIFILE" && touch "$SRIFILE"
+
+printf "{ \"_comment\": \"@generated by pkgs/development/tools/phase/update.sh\"\n" >> "$SRIFILE"
+
+# Download sha256sums for all targets from Phase CLI's GitHub releases
+for arch in "${ARCHS[@]}"; do
+    IFS=: read -r arch_name arch_target <<< "$arch"
+
+    # Get the SHA256 sum directly from the release assets
+    checksum_url="https://github.com/$REPO/releases/download/v${VERSION}/phase_cli_${arch_target}_${VERSION}.zip.sha256"
+    sha256sum=$(curl -sL "$checksum_url" | cut -d' ' -f1)
+
+    # Convert the SHA256 Hex hash to base64 for SRI
+    srihash="$(nix hash to-sri --type sha256 "$sha256sum")"
```

Comment:
```suggestion
    srihash="$(nix hash convert --hash-algo sha256 "$sha256sum")"
```

Reply:
Tried using convert. It was returning an error:
```
copying path '/nix/store/xcind1c9z66qg6g8xvj8ckz8dnp6r80q-binutils-wrapper-2.41' from 'https://cache.nixos.org'...
copying path '/nix/store/q6afjcz9h6ybcrai8w4z4k6ld7yqi9nd-gcc-wrapper-13.2.0' from 'https://cache.nixos.org'...
copying path '/nix/store/61h5fgmxl3biljwbhxvr55qkxsp44yla-stdenv-linux' from 'https://cache.nixos.org'...
Latest Phase CLI release: 1.18.6
error: 'convert' is not a recognised command
```

Please advise if I need to change something else.

Reply:
Are you using Lix? If so it might not exist yet there

Reply:
Thanks for replying. I'm using the latest `nixos/nix` docker image.

Reply:
> Thanks for replying. I'm using the latest `nixos/nix` docker image.

Try running `nix --version` — it would not work if the Nix version is under 2.20. I don't use Docker so I don't know what the default is over there (though I can wager a guess that it's probably 2.18), but Nixpkgs itself uses Nix 2.24 by default so you *should* use `nix hash convert` here.

Reply:
> Try running nix --version — it would not work if the Nix version is under 2.20. I don't use Docker so I don't know what the default is over there (though I can wager a guess that it's probably 2.18), but Nixpkgs itself uses Nix 2.24 by default so you should use nix hash convert here.
```
# nix --version
nix (Nix) 2.25.2
```